src/modules/core/filter_pillar_echo.c

```diff
@@ -59,7 +59,7 @@ typedef struct
     mlt_rect rect;
 } scale_sliced_desc;
 
-static int scale_sliced_proc(int id, int index, int jobs, void *data)
+static int scale_sliced_proc_rgb32(int id, int index, int jobs, void *data)
```

Comment:
This is rgb24 not 32

Reply:
Good catch. It is actually rgba32. Fixed now.